deploy/utils.sh

```diff
@@ -479,3 +475,43 @@ get_logs() {
     get_conjur_logs_container
     get_app_logs_container
 }
+
+# TODO: remove this and use get_pods_info instead
+cli_get_pods_test_env() {
+  $cli_with_timeout "get pods --namespace=$APP_NAMESPACE_NAME --selector app=test-env --no-headers"
+}
+
+# Return the pods information of a given namespace and app name without the headers
+# For example: 'test-app-5-fab52b20-0 secret-provider-0 1/1 Running 1 20m'
+get_pods_info() {
+  local namespace=$1
+  local app_name=$2
+
+  $cli_with_timeout get pods \
+    --namespace=${namespace} \
+    --selector app=${app_name} \
+    --no-headers
+}
+
+# TODO: use this method for all cases that need the pod name
+# Return the pod name of a given namespace and app name
+# For example: 'secret-provider-0'
+get_pod_name() {
+  local namespace=$1
+  local app_name=$2
+
+  pod_name=$(
+    $cli_with_timeout get pods \
+      --namespace=${namespace} \
+      --selector app=${app_name} \
+      -o jsonpath='{.items[].metadata.name}'
+  )
+
+  if [[ -z ${pod_name:-""} ]]; then
+    echo "Unable to find ${app_name} pod in namespace ${namespace} - aborting."
+    eval "${cli_with_timeout}" describe pods --namespace="${namespace}"
```

Comment:
I think there's no need to use `eval` here.
call `$cli_with_tiomeout`, like above.